tests/lib/types/types.test.ts

```diff
@@ -42,9 +42,7 @@ import { ESLintRules } from "eslint/rules";
 import { Linter as ESLinter } from "eslint/universal";
 import {
 	builtinRules,
-	FileEnumerator,
 	FlatESLint,
```

Comment:
```suggestion
```

Let's also remove this type and its usages in the type tests.

https://github.com/eslint/eslint/blob/6139c7eaf12ac0f7242fb4c790f37329ef343f2d/tests/lib/types/types.test.ts#L45

https://github.com/eslint/eslint/blob/6139c7eaf12ac0f7242fb4c790f37329ef343f2d/tests/lib/types/types.test.ts#L2069